### ISSUE TITLE
In Wikimedia ZIMs produced since October, dark theme cannot be switched without changing the OS theme

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-windows#795]
Unfortunately, new Wikipedia ZIMs produced since October have responsive dark-light themes based on the OS's dark-light settings, but do not provide any standardized way of detecting this support. This creates issues if the user wants to display a page in light theme (overriding the OS settings), because there is no way to switch off the dark theme without changing the entire OS setting.

This is a serious bug and useability issue, with accessibility concerns as well for users with vision issues, so it is high priority to "fix".

It seems there is no official method to detect whether a ZIM has been built with this behaviour, so any temporary solution will have to be a hack that inspects the HTML/CSS in the ZIM to find out the effective colour scheme that has been applied. See https://github.com/openzim/mwoffliner/issues/1136 for discussion of this (unresolved).

**Comment by Jaifroid** [comment]
See also https://github.com/kiwix/kiwix-js/issues/1376 for the upstream KJS issue.
